remoteit/desktop
# [DESK-702] deep links

### Description


Implement deep link connections

### Test plan



1. Sign in
2. Find the Service ID of a service
3. Create the deep link url: `remoteit://connect/<serviceId>`
4. Open a browser and type this in the url
5. App should open and connect
6. Quit the app
7. Re-test url
8. Sign out of the app
9. Re-test url

### Ticket



<https://remoteit.atlassian.net/browse/DESK-702>
